go.mod

```diff
@@ -2,11 +2,13 @@ module github.com/prometheus/snmp_exporter
 
 go 1.23.0
 
+toolchain go1.24.1
```

Comment:
@SuperQ here comes toolchain again - how do we want to deal with this, you explicitly removed it here https://github.com/prometheus/snmp_exporter/pull/1396/commits/586e2ca45d55e2375e0b2611237d68925c4a97b5